cfgov/unprocessed/css/nav-secondary.less

```diff
@@ -58,67 +50,44 @@
       codenotes:
         - ".nav-secondary_link__current"
       notes:
-        - "Current links are hidden on small screens and the link button is
+        - "Current links are hidden on small screens and the menu button is
            revealed."
-        - "Currentl links have a left green border on large screens."
-    - name: Link button modifier
-      markup: |
-        <button class="nav-secondary_link nav-secondary_link__button">
-            Button nav link
-        </button>
-      codenotes:
-        - ".nav-secondary_link__button"
-      notes:
-        - "This modifier is only visible on smaller screens."
-        - "The button link modifier is used to convey the current link for small
-           screens. It also acts as a toggle button when mixing this pattern
-           with the expandable pattern."
-        - "The inline style is for demonstration purposes only and is needed
-           because the modifier hides the element at larger screen size.
-           Please do not use it in production."
-    - name: disabled link modifier
-      markup: |
-        <a class="nav-secondary_link nav-secondary_link__disabled">
-            Disabled nav link
-        </a>
-      codenotes:
-        - ".nav-secondary_link__disabled"
-      notes:
-        - "Gives the link disabled link styles,
-           for when there is no href value, for instance."
+        - "Current links have a left green border on all screens."
     - name: Secondary nav mixed with cf-expandable
       markup: |
         <nav class="nav-secondary expandable">
             <div class="expandable_header">
                 <button class="expandable_target
-                        nav-secondary_link
-                        nav-secondary_link__button">
+                               nav-secondary_button">
                     <span class="expandable_header-left">
-                        Item 2
+                        Nav Button
                     </span>
                     <span class="expandable_header-right">
                         <span class="expandable_cue-open">
-                            <span class="cf-icon cf-icon-down cf-icon__green"></span>
+                            <span class="cf-icon
+                                         cf-icon-down"></span>
                         </span>
                         <span class="expandable_cue-close">
-                            <span class="cf-icon cf-icon-up cf-icon__green"></span>
+                            <span class="cf-icon
+                                         cf-icon-up"></span>
                         </span>
                     </span>
                 </button>
             </div>
             <div class="expandable_content">
                 <ul class="nav-secondary_list">
```

Comment:
Entirely forgot about this but finally fixed it here -> https://github.com/cfpb/cfgov-refresh/pull/1245